### PR TITLE
Add Jellyfin Docs

### PR DESCRIPTION
**Pull request by Dominion5254**

### CODE REVIEW

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+.. contents::
+  :depth: 2 
+  :local:
+
+.. _jellyfin-webapp:
+
+After setup has been completed per the Jellyfin Service instructions on your Server, there are many ways to connect to and use your Jellyfin server.
+
+.. note:: While the Jellyfin server is accessable over tor for many clients, connecting over LAN via your Jellyfin .local URL is recommended for the best streaming experience.
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst:15]:
```suggestion
.. note:: While the Jellyfin server is accessable over tor for many clients, connecting over LAN via your Jellyfin `.local` URL is recommended for the best streaming experience.
```

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Web Application
+---------------
+
+1. Launch the Jellyfin Web Application from the desired interface (.local recommended). If you are having trouble connecting, make sure your browser and OS have been setup to connect to :ref:`LAN <connecting-lan>` or :ref:`Tor <connecting-tor>` depending on the interface you are trying to connect to.
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst:20]:
```suggestion
1. Launch the Jellyfin Web Application from the desired interface (`.local` recommended). If you are having trouble connecting, make sure your browser and OS have been setup to connect to :ref:`LAN <connecting-lan>` or :ref:`Tor <connecting-tor>` depending on the interface you are trying to connect to.
```

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+-----------
+
+Android
+=======
+
+Before proceeding, make sure your Android device has been setup to connect over :ref:`LAN <lan-android>`. If you are connecting over tor, you will need to :ref:`Setup Tor <tor-android>`.
+
+1. Visit the app store of your choice and download the Jellyfin app.
+
+2. Upon entering the app, you will be prompted for a host; paste the URL (.local recommended) of your Jellyfin server in this field and you should see your media splash page. If you are getting connection errors, make sure you are connected to the same LAN and a VPN is not in use on your Android device.
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst:32]:
```suggestion
2. Upon entering the app, you will be prompted for a host; paste the URL (`.local` recommended) of your Jellyfin server in this field and you should see your media splash page. If you are getting connection errors, make sure you are connected to the same LAN and a VPN is not in use on your Android device.
```

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+iOS
+===
+
+Before proceeding, make sure your Apple device has been setup to connect over :ref:`LAN <lan-ios>`. If you are connecting over tor, you will need to :ref:`Setup Tor <tor-ios>`.
+
+1. Open the Apple app store and download the Jellyfin app.
+
+2. Upon entering the app, you will be prompted for a host; paste the URL (.local recommended) of your Jellyfin server in this field and you should see your media splash page. If you are getting connection errors, make sure you are connected to the same LAN and a VPN is not in use on your Apple device.
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst:41]:
```suggestion
2. Upon entering the app, you will be prompted for a host; paste the URL (`.local` recommended) of your Jellyfin server in this field and you should see your media splash page. If you are getting connection errors, make sure you are connected to the same LAN and a VPN is not in use on your Apple device.
```

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+------------
+
+MacOS
+=====
+
+Before proceeding, make sure your Apple device has been setup to connect over :ref:`LAN <lan-mac>`.
+
+1. Download the .dmg file for `Jellyfin Media Player <https://github.com/jellyfin/jellyfin-media-player/releases>`_
+
+2. Upon entering the app, you will be prompted for a host; paste the .local URL (.onion is not supported) of your Jellyfin server in this field and you should see your media splash page. If you are getting connection errors, make sure you are connected to the same LAN and a VPN is not in use on your Apple device.
```
Comment (kn0wmad) [inline site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst:53]:
```suggestion
2. Upon entering the app, you will be prompted for a host; paste the `.local` URL (`.onion` is not supported) of your Jellyfin server in this field and you should see your media splash page. If you are getting connection errors, make sure you are connected to the same LAN and a VPN is not in use on your Apple device.
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst (hunk 1 above)
Comment (Dominion5254) [reply site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst]: Updated

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst (hunk 2 above)
Comment (Dominion5254) [reply site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst]: Updated

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst (hunk 3 above)
Comment (Dominion5254) [reply site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst]: Updated

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst (hunk 4 above)
Comment (Dominion5254) [reply site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst]: Updated

File: site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst (hunk 5 above)
Comment (Dominion5254) [reply site/source/user-manual/service-guides/jellyfin/jellyfin-clients.rst]: Updated

**Review by kn0wmad — APPROVED** [review]

**Review by kn0wmad — APPROVED** [review]
Good catch

**Pull request merged**